litellm/llms/bedrock/chat/converse_transformation.py

```diff
@@ -602,6 +602,33 @@ def _transform_thinking_blocks(
                 thinking_blocks_list.append(_thinking_block)
         return thinking_blocks_list
 
+    def _transform_usage(self, usage: ConverseTokenUsageBlock) -> Usage:
+        input_tokens = usage["inputTokens"]
+        output_tokens = usage["outputTokens"]
+        total_tokens = usage["totalTokens"]
+        cache_creation_input_tokens: int = 0
+        cache_read_input_tokens: int = 0
+
+        if "cacheReadInputTokens" in usage:
+            cache_read_input_tokens = usage["cacheReadInputTokens"]
+            input_tokens += cache_read_input_tokens
```

Comment:
This isn't how its returned in Bedrock - the input tokens are just the ones that weren't cached.